test/sslapitest.c

```diff
@@ -9395,7 +9397,7 @@ static int test_sni_tls13(void)
         goto end;
     /* Require TLSv1.3 as a minimum */
     if (!TEST_true(create_ssl_ctx_pair(libctx, TLS_server_method(),
-                                       TLS_client_method(), TLS1_3_VERSION, 0,
+                                       TLS_client_method(), 0, 0,
```

Comment:
Why this change?

Reply:
I have no idea!! I don't remember changing that and it doesn't look like a correct change either. Although in practice it probably doesn't make a difference because this code won't be compiled unless we have usable TLSv1.3 and we will always use it by default. I'll fix it.